src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -2733,7 +2733,7 @@ protected function generateTree($path, $intMargin, $mount=false, $blnProtected=t
 					}
 					catch (RuntimeException $e)
 					{
-						$thumbnail .= '<br><p class="broken-image" style="margin:0 0 2px -18px">Broken image!</p>';
+						$thumbnail .= '<br><p class="broken-image" style="margin:0 0 2px -18px;max-width:400px;max-height:50px;width:auto;height:auto">Broken image!</p>';
```

Comment:
Why adding the style to the `<p>` tag too?

Reply:
Search and replace error ;). I'll fix it. I'll also add something similar for the important part preview (just in case).